cellprofiler/modules/classifyobjects.py

```diff
@@ -124,11 +126,12 @@ def create_settings(self):
         # The object for the contrasting method
         #
         self.object_name = cps.ObjectNameSubscriber(
-                "Select the object name", cps.NONE, doc="""
-            Choose the object that you want to measure from the list.
-            This should be an object created by a previous module such as
-            <b>IdentifyPrimaryObjects</b>, <b>IdentifySecondaryObjects</b>, or
-            <b>IdentifyTertiaryObjects</b>.""")
+                "Select the object name", cps.NONE, doc="""\
+Choose the object that you want to measure from the list. This should be
+an object created by a previous module such as
+**IdentifyPrimaryObjects**, **IdentifySecondaryObjects**, or
+**IdentifyTertiaryObjects**.
```

Comment:
Sorry, can you add **Watershed** to this list too? Didn't realize there were two lists!